packaging/windows/install_script.nsis

```diff
@@ -194,7 +194,7 @@ Section "uninstall"
     Pop $0
 
     # Remove files
-    delete $INSTDIR\agent-windows-amd64.exe
+    delete $INSTDIR\grafana-agent-windows-amd64.exe
```

Comment:
My gut says we should delete the old name here too.

Reply:
It does not https://nsis.sourceforge.io/Reference/Delete `The error flag is not set from trying to delete a file that does not exist. `

Reply:
👍 thanks!